cogs/librarian/librarian.py

```diff
@@ -1,9 +1,9 @@
 import base64
-import requests
 import hashlib
 from datetime import date
 
 from discord.ext import commands
+import aiohttp
```

Comment:
Please, use the same ordering the rest of the project does -- general libraries are in the first import group and they are sorted alphabetically.

```py
import aiohttp
import base64
import hashlib
...
```